src/i18n/en-US/systemProfile.ts

```diff
@@ -5,9 +5,39 @@ const systemProfile = {
     info: 'Have a new system or application?',
     button: 'Start an intake request'
   },
+  navigation: {
+    home: 'System Home',
+    details: 'System Details',
+    'team-and-contract': 'Team and Contract',
```

Comment:
The reason these are no camel-cased is because the keys directly correspond to the url params that identify which components to render.  This could have been done a lot of different ways, but I found this was the most succinct.

Reply:
Using a key to map params, components, and translations relieved the need for lots of filtering and searching through arrays.